lib/billy/railtie.rb

```diff
@@ -1,9 +1,7 @@
-module Billy
-  class Railtie < Rails::Railtie
-    railtie_name 'billy'
+# Deprecated
+require 'billy/browsers/capybara'
```

Comment:
Railties are not specific to Capybara.  I think this should remain generic to be used across the board.  I see this assumed tie to Capybara in a few places.